### DISCUSSION TITLE
Time Manager Reports User

### DISCUSSION THREAD

**Discussion by tsalzsieder** [te-online/timemanager#138] [Q&A]
Lieber Thomas, 

seit kurzer Zeit bin ich nicht mehr in der Lage die reports für andere runterzuladen (wer aus meinem Team hat wann was gearbeitet). Dies wäre aber drigned notwendig. Wurde dieses Feature entfernt? Falls nicht, wie kann ich die reports für andere runterladen als Admin? Speziell kann ich nicht mehr die User auswählen, sondern wenn dann nur nach Projekt filtern. Nicht aber nach User.

Vielen Dank für deine Hilfe!

Beste Grüsse 
Tom

**Comment by te-online** [comment]
Hey Tom 👋 

Das Feature wurde nicht entfernt, aber vielleicht funktioniert es nicht mehr wie zuvor?
Wie im Screenshot zu sehen, müsste auf der "Reports" Seite eine Suchbox für "Created by" sein. Dort kannst du nach Anzeigenamen von Nutzern suchen. Wählst du einen Nutzer aus, sollten nur die Einträge dieses Benutzer angezeigt werden.

Lass mich gerne wissen, ob das funktioniert, wie vorgesehen.

![Screenshot 2023-08-26 at 14 25 52](https://github.com/te-online/timemanager/assets/4127167/1609a381-958d-4632-98f9-09c5ec0a76c9)
